src/ints/mouse.cpp

```diff
@@ -124,77 +114,60 @@ typedef struct MouseEvent {
     bool request_vmm = false; // if virtual machine mouse needs an event
     bool request_dos = false; // if DOS mouse driver needs an event
 
-    MouseEventId id             = MouseEventId::NotDosEvent;
-    uint8_t mask                = 0;
+    bool dos_moved   = false;
+    bool dos_button  = false;
+    bool dos_wheel   = false;
     MouseButtons12S buttons_12S = 0;
 
-    uint8_t aggregate_mask = 0; // to check if button events can be aggregated
-
-    MouseEvent() = default;
-    MouseEvent(MouseEventId id) : id(id), mask(static_cast<uint8_t>(id)) {}
-
 } MouseEvent;
 
 static class MouseQueue final {
 public:
     MouseQueue();
 
-    void AddEvent(MouseEvent &event);
-    void FetchEvent(MouseEvent &event);
+    void AddEvent(MouseEvent &ev);
+    void FetchEvent(MouseEvent &ev);
     void ClearEventsDOS();
     void StartTimerIfNeeded();
     void Tick();
 
     struct { // intial value of delay counters, in milliseconds
-        uint8_t ps2_ms             = 5;
-        uint8_t dos_button_ms      = 1;
-        uint8_t dos_moved_wheel_ms = 5;
+        uint8_t ps2_ms = 5;
+        uint8_t dos_ms = 5;
```

Comment:
Maybe we can initialize using the top-level latency now?

``` c++
   struct { // intial value of delay counters, in milliseconds
        uint8_t ps2_ms = max_delay_ms;
        uint8_t dos_ms = max_delay_ms;
```

Reply:
Indeed, it makes more sense - changed.